_includes/money-bar.html

```diff
@@ -5,7 +5,7 @@
 {% when 'percent' %}
 {% assign display = width | floor | append: '%' %}
 {% else %}
-{% assign display = include.value | times: 100 | money %}
+{% assign display = include.value | friendly_number | prepend: '$' %}
```

Comment:
Should this one be dollars too?

Reply:
This one is different, mainly to save space. This will be rendered as `$10M` instead of `$10,000,000`.